src/main/java/eu/dissco/backend/repository/UserRepository.java

```diff
@@ -0,0 +1,77 @@
+package eu.dissco.backend.repository;
+
+import static eu.dissco.backend.database.jooq.Tables.NEW_USER;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.backend.database.jooq.tables.records.NewUserRecord;
+import eu.dissco.backend.domain.User;
+import java.time.Instant;
+import java.util.Map;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.jooq.DSLContext;
+import org.jooq.Record;
+import org.jooq.TableField;
+import org.springframework.stereotype.Repository;
+
+@Slf4j
+@Repository
+@RequiredArgsConstructor
+public class UserRepository {
+
+  private final DSLContext context;
+  private final Map<String, TableField<NewUserRecord, String>> attributeMapping = Map.of(
+      "firstName", NEW_USER.FIRST_NAME,
+      "lastName", NEW_USER.LAST_NAME,
+      "email", NEW_USER.EMAIL,
+      "orchid", NEW_USER.ORCHID,
```

Comment:
"Orchid" = the plant
"Orcid" = open researcher and contributer id
You probably want to change this

Reply:
:+1:

Reply:
Fixed